tgui/packages/tgui/interfaces/common/SortButton.js

```diff
@@ -0,0 +1,33 @@
+import { useLocalState } from '../../backend';
+import {
+  Button,
+  Icon,
+  Table,
+} from '../../components';
+
+export const SortButton = (properties, context) => {
+  const [sortId, setSortId] = useLocalState(context, 'sortId', 'name');
+  const [sortOrder, setSortOrder] = useLocalState(context, 'sortOrder', true);
+  const { id, children } = properties;
+  return (
+    <Table.Cell>
```

Comment:
Don't include table elements in this component. It's best to keep this as close to its intended purpose as possible.

Reply:
This is how it was before for both security and the account console. Changing this produces this result. 
![image](https://user-images.githubusercontent.com/91113370/205453733-f12fdb62-eca4-45f7-8e4f-0e28012f489f.png)
I'm just abstracting this to its own file here, this is how it was before and would require its own cells around each of them, meaning I would need to change both security and accounting files, which I feel is out of scope.